tests/Backend/Mixed/SharingToSpecificProjectsTest.php

```diff
@@ -86,9 +87,28 @@ public function testShareBucketToProject($backend)
             $project = reset($sharedBucket['sharingParameters']['projects']);
             $this->assertEquals($targetProject['id'], $project['id']);
             $this->assertEquals($targetProject['name'], $project['name']);
+
+            $response = $this->_client2->getSharedBucketDetail($sourceProject['id'], $bucketId);
+
+            $this->assertNotEmpty($response);
+            $this->assertSame($bucketId, $response['id']);
+
+            try {
+                $this->clientAdmin2InSameOrg->getSharedBucketDetail($targetProject['id'], $bucketId);
+            } catch (ClientException $e) {
+                $this->assertEquals('storage.buckets.notFound', $e->getStringCode());
+                $this->assertEquals(404, $e->getCode());
+            }
+
+            try {
+                $this->clientAdmin3InOtherOrg->getSharedBucketDetail($targetProject['id'], $bucketId);
+            } catch (ClientException $e) {
+                $this->assertEquals('storage.buckets.notFound', $e->getStringCode());
+                $this->assertEquals(404, $e->getCode());
+            }
         }
 
-        $this->assertCount(2, $this->_client->listSharedBuckets());
+        $this->assertCount(2, $this->_client2->listSharedBuckets());
```

Comment:
proč je tady tahle změna? V testu není změna která by to původní změnila. Takže by to mělo dál fungovat. Takže klidně může přibýt assert na client2 ale ten na client1 by furt měla fungovat...

Reply:
Ono je to na riadku 57 a 58 asi trochu blbo vidiet ale zmenil som targetProject na ten co ma `_client2` a to z toho dovodu ze chcem otestovat ze to skutocne na inom "ucte" ziskam a na inych nie, myslim si ze ten target na `client2` tam mal uz byt predtim ale asi sa to prehliadlo

Reply:
Jasně, vidím OK.